dns.go

```diff
@@ -109,9 +109,13 @@ func probeDNS(ctx context.Context, target string, module Module, registry *prome
 		probeDNSAdditionalRRSGauge.Set(float64(numAdditional))
 	}()
 
+	if module.DNS.QueryName == "" {
```

Comment:
This should be part of the config validation, we shouldn't accept a config with this missing.